src/main/java/Avalon.java

```diff
@@ -0,0 +1,136 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+public class Avalon {
+    public static void main(String[] args) {
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+        Scanner scanner = new Scanner(System.in);
+        //greet
```

Comment:
Do we have separate logical units here (as suggested by the comment)? In that case, a blank line might improve readability here. There were other instances in the code base as well which could benefit from such improved readability.